central/graphql/resolvers/deployments.go

```diff
@@ -743,12 +743,11 @@ func (resolver *deploymentResolver) unresolvedAlertsExists(ctx context.Context,
 	if err != nil {
 		return false, err
 	}
-	q.Pagination = &v1.QueryPagination{Limit: 1}
-	results, err := resolver.root.ViolationsDataStore.Search(ctx, q)
+	count, err := resolver.root.ViolationsDataStore.Count(ctx, q)
```

Comment:
I'm cool with this change, but what if we added Exists for a query? this would stop very early then and be extremely cheap

Reply:
Fair enough. Likely cannot accommodate it before my PTO.

Reply:
Yeah, can you just cut a ticket for it? Feel free to ship in the meantime